MFE/dashboard/src/view/widget/wizard/wizard.jsx

```diff
@@ -89,7 +139,7 @@ const Wizard = ({ initialValues, ...props }) => {
       <div className={classes.root}>
         <Stepper activeStep={page} orientation='vertical' connector={<StepLine />}>
           {steps.map(({ label, key }) => (
```

Comment:
What do you think of using the `index` of this map to navigate through steps. This way you don't need the `stepperNavigator` function.